crates/store/re_types/definitions/rerun/blueprint/datatypes/component_column_selector.fbs

```diff
@@ -15,7 +15,7 @@ table ComponentColumnSelector (
   /// The name of the component.
   ///
   /// This acts as the component name in the context of a given `entity_path`
-  /// An example for this would be `Points3D:positions`, for the `positions` field in the `Points3D` archetype.
+  /// An example for this would be `Points3D:positions`, for the `positions` field in the [archetypes.Points3D] archetype.
```

Comment:
ah thx, ci really ought to do warnings-as-errors on this. I think I filed a ticker for that somewhere :)